src/redux/reducer/reducer.js

```diff
@@ -0,0 +1,25 @@
+import { SHOW_CARD, HIDE_CARD } from "../constants";
+import { usersList } from "../../data/userList";
+
+const intialState = {
+  userData: usersList,
```

Comment:
This state variable should be named `usersData`, because it holds multiple user's data. Or to simplify things you could have just called it `usersList`.